runtime/themes/material_deep_ocean.toml

```diff
@@ -0,0 +1,123 @@
+# Material Theme <https://material-theme.com/> for Helix Editor
+
+# Syntax Highlighting
+
+"type" = "purple"
+
+"constructor" = "blue"
+
+"constant" = "yellow"
+
+"string" = "green"
+"string.regexp" = "yellow"
+"string.special" = "blue"
+
+"comment" = { fg = "comment" }
+
+"variable" = "text"
+"variable.parameter" = { fg = "orange" }
+"variable.builtin" = "yellow"
+
+"label" = "orange"
+
+"punctuation" = "cyan"
+
+"keyword" = "purple"
+"keyword.storage" = "cyan"
+
+"operator" = "cyan"
+
+"function" = "blue"
+"function.macro" = "cyan"
+
+"tag" = "red"
+"attribute" = "purple"
+
+"namespace" = { fg = "yellow" }
+
+"special" = "cyan"
+
+"markup.heading.marker" = { fg = "cyan", modifiers = ["bold"] }
+"markup.heading.1" = "cyan"
+"markup.heading.2" = "red"
+"markup.heading.3" = "green"
+"markup.heading.4" = "yellow"
+"markup.heading.5" = "blue"
+"markup.heading.6" = "orange"
+"markup.list" = "purple"
+"markup.bold" = { modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
+"markup.strikethrough" = { modifiers = ["crossed_out"] }
+"markup.link.url" = { fg = "green", modifiers = ["underlined"] }
+"markup.link.text" = "blue"
+"markup.raw" = "text"
+
+"diff.plus" = "green"
+"diff.minus" = "red"
+"diff.delta" = "blue"
+
+# User Interface
+
+"ui.background" = { bg = "bg", fg = "text" }
+"ui.text" = { fg = "text" }
+
+"ui.statusline" = { bg = "bg", fg = "text" }
+"ui.statusline.inactive" = { bg = "bg", fg = "disabled" }
+"ui.statusline.normal" = { bg = "accent", fg = "text" }
+"ui.statusline.insert" = { bg = "green", fg = "text" }
+"ui.statusline.select" = { bg = "purple", fg = "text" }
+
+
+"ui.selection" = { bg = "selection" }
+
+"ui.virtual" = { fg = "gray" }
+
+"ui.virtual.indent-guide" = { fg = "gray" }
```

Comment:
You may want to set `ui.virtual.ruler` to `{ bg = "gray" }`. Otherwise the rulers will only show up when text overlaps with them. You can use `:set rulers [80]` to experiment

Reply:
Isn't "virtual" = { fg = "gray" } means all the keys in "virtual" will be with foreground gray?

Reply:
Oh you mean the background, Sorry